src/artifact_providers/__tests__/base.test.ts

```diff
@@ -9,33 +8,44 @@ describe('parseFilterOptions', () => {
     expect(parsedFilters).not.toHaveProperty('excludeNames');
   });
 
-  test('undefined properties', () => {
-    const rawFilters: RawFilterOptions = {
-      includeNames: undefined,
-      excludeNames: undefined,
-    };
-    const parsedFilters = parseFilterOptions(rawFilters);
-    expect(parsedFilters).not.toHaveProperty('includeNames');
-    expect(parsedFilters).not.toHaveProperty('excludeNames');
-  });
+  test.each([
+    [undefined, undefined],
+    [undefined, '/exclude/'],
+    [undefined, new RegExp('exclude')],
+    ['/include/', undefined],
+    [new RegExp('include'), undefined],
+    ['/include/', '/exclude/'],
+    ['/include/', new RegExp('exclude')],
+    [new RegExp('include'), '/exclude/'],
+    [new RegExp('include'), new RegExp('exclude')],
```

Comment:
why not the short form of regular expressions?

```suggestion
    [undefined, /exclude/],
    ['/include/', undefined],
    [/include/, undefined],
    ['/include/', '/exclude/'],
    ['/include/', /exclude/],
    [/include/, '/exclude/'],
    [/include/, /exclude/],
```